detox/e2e/support/server_api/common.js

```diff
@@ -2,7 +2,20 @@
 // See LICENSE.txt for license information.
 
 export const getResponseFromError = (err) => {
-    const {response: {data, status}} = err;
+    const {response} = err;
+    if (!response) {
+        const message = `No response from server at "${err.config.baseURL}".
+If testing against a server other than the default local instance, you may set the server URL via "SITE_URL" environment variable.
+`;
+
+        // Throw an error instead of failing silently
+        throw new Error(message);
+    }
+
+    const {data, status} = response;
+
+    // Explicitly print out response data from server for ease of debugging
+    console.warn(data); // eslint-disable-line no-console
```

Comment:
Direct server API request is failed silently since there might be some instance where we need to validate an error. In case of error, explicitly log the response data (as warning) for ease of debugging.
<img width="459" alt="Screen Shot 2020-09-28 at 6 18 37 PM" src="https://user-images.githubusercontent.com/5334504/94421767-9d52ed80-01b8-11eb-8406-6b97a46a988a.png">